src/main/cpp/hw/DragonTalonFX.cpp

```diff
@@ -151,33 +158,16 @@ void DragonTalonFX::ConfigMotorSettings(InvertedValue inverted,
 }
 void DragonTalonFX::SetPIDConstants(int slot, double p, double i, double d, double f)
 {
-	if (slot == 0)
-	{
-		Slot0Configs slot0;
-		slot0.kP = p;
-		slot0.kI = i;
-		slot0.kD = d;
-		slot0.kV = f;
-		m_talon.GetConfigurator().Apply(slot0);
-	}
-	else if (slot == 1)
-	{
-		Slot1Configs slot1;
-		slot1.kP = p;
-		slot1.kI = i;
-		slot1.kD = d;
-		slot1.kV = f;
-		m_talon.GetConfigurator().Apply(slot1);
-	}
-	else if (slot == 2)
-	{
-		Slot2Configs slot2;
-		slot2.kP = p;
-		slot2.kI = i;
-		slot2.kD = d;
-		slot2.kV = f;
-		m_talon.GetConfigurator().Apply(slot2);
-	}
+	ControlData controlInfo{ControlModes::CONTROL_TYPE::VOLTAGE,
```

Comment:
Does this mean that anytime we set PID constants, a motor will go to voltage control?  Maybe I'm not understanding correctly, but if we set this to slot 0, then when trying to set a motor it may not be the control mode we assume (percent output, etc.)

Reply:
Normally we aren't going to use this one.  It is the overridden method for the interface.  We should be using SetControlConstants in the normal flow for this.

We may want to strip down the interface some and just get rid of this method.   I may do this in the next iteration just for clarity sake.